### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by maniksharma424**

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export const MAIN_CONTAINER = "bg-gradient-to-t from-[#32235c] via-[#48296d] via-[#602e7d] via-[#7a328b] to-[#953498] text-white"
 
-export const CONTAINER = "container max-w-full h-[800px] flex flex-col justify-around items-center"
+export const CONTAINER = "container max-w-full sm:w-2/5 h-[800px] sm:ml-[400px] flex flex-col justify-around items-center"
```
Comment (arjunbharti) [inline src/constants.js:3]: don't use capitals here for classnames, have consistency across codebase!

File: src/pages/Chart.jsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  console.log("chartjs");
+  useEffect(() => {
+    const getChartData = async () => {
+      const response = await fetch(
+        `https://www.alphavantage.co/query?function=TIME_SERIES_DAILY_ADJUSTED&symbol=${item["1. symbol"]}&apikey=` +
+          ALPHA_VANTAGE_KEY
+      );
+      const jsonResponse = await response.json();
+      const jsonData = await jsonResponse["Time Series (Daily)"];
+      console.log("chart data is " + jsonData);
```
Comment (arjunbharti) [inline src/pages/Chart.jsx]: remove logs from the codebase

File: src/pages/Chart.jsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      const labelValues = Object.keys(jsonData).map((key) => formatDate(key));
+
+      setChartData({
+        dataValues: dataValues,
+        labelValues: labelValues,
+      });
+    };
+    getChartData();
+  }, []);
+  console.log(chartData);
```
Comment (arjunbharti) [inline src/pages/Chart.jsx]: remove log

File: src/pages/Chart.jsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        pointHoverBorderColor: "rgba(255, 99, 132, 1)",
+        pointHoverRadius: 2,
+        lineTension: 0.4,
+      },
+    ],
+  };
+  const options = {
+    // plugins: {
+    //   legend: true,
+    // },
```
Comment (arjunbharti) [inline src/pages/Chart.jsx]: remove unused code

File: src/pages/StockModal.jsx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+        `https://www.alphavantage.co/query?function=GLOBAL_QUOTE&symbol=${item["1. symbol"]}&apikey=` +
+          ALPHA_VANTAGE_KEY
+      ).then((res) => res.json());
+      const info = await response["Global Quote"];
+      setData(info);
+    };
+    getPrice();
+
+  }, []);
+  console.log(data);
```
Comment (arjunbharti) [inline src/pages/StockModal.jsx]: remove log

File: src/pages/StockModal.jsx [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+const background = useSelector(store=>store.dashboardSlice.modalBg)
+ 
+if(data === undefined) return <p>hi</p>
```
Comment (arjunbharti) [inline src/pages/StockModal.jsx]: not a good practice to conditionally check using `undefined`

File: src/pages/StockModal.jsx [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+
+else return (
+    <motion.div
```
Comment (arjunbharti) [inline src/pages/StockModal.jsx]: use ternary operator and avoid if else to maintain es6 syntax

File: src/pages/UserInfo.jsx [hunk 8]
```diff
@@ -1,32 +1,51 @@
-import React from "react";
-import { useSelector } from "react-redux";
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
 import { motion } from "framer-motion";
+import Link from "next/link";
+import { setDay, setModalDefault, setModalNight, setNight, setTextDefault, setTextNight } from "@/slices/dashboardSlice";
 const UserInfo = () => {
-
+  const [isDay, setIsDay] = useState(true);
```
Comment (arjunbharti) [inline src/pages/UserInfo.jsx:7]: can you explain this piece of code here?
and why you kept the state as true by default?

File: src/pages/UserInfo.jsx [hunk 9]
```diff
@@ -1,32 +1,51 @@
...
   const date = new Date();
-  const addedBookmarks = useSelector((store) => store?.bookmarks?.bookmarks);
+  const dispatch = useDispatch();
 
+console.log('user info');
```
Comment (arjunbharti) [inline src/pages/UserInfo.jsx]: remove log

File: src/utilities.js [hunk 10]
```diff
@@ -22,6 +22,7 @@ const getSuggestionResults = async (text, setSuggestions) => {
   const json = await response.json().catch((err) => {
     throw new Error();
   });
+
   if (json.bestMatches) return setSuggestions(json?.bestMatches);
   else return null;
```
Comment (arjunbharti) [inline src/utilities.js:31]: ternary >>>> if else

**Review by arjunbharti — CHANGES_REQUESTED** [review]
General feedback:

1. add a proper name for your branch and no CAPSLOCK please
2. add a proper description in the PR so that the reviewer get some context of what he/she is reviewing
3. check for logs before pushing your code
4. avoid using if else and try adapting es6 syntax (ternary >>>> if else)

File: src/pages/UserInfo.jsx (hunk 8 above)
Comment (maniksharma424) [reply src/pages/UserInfo.jsx:7]: this is for the day and night theme and by default, it is set to day

**Pull request closed without merging**